target-libretro/libretro.cpp

```diff
@@ -613,18 +689,6 @@ void retro_reset(void) {
   SuperFamicom::system.reset();
 }
 
-void retro_run(void) {
```

Comment:
Don't move functions around, that just makes it harder to find the real changes. Static functions can have prototypes, do that instead.